drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -634,6 +648,20 @@ function DragAndDropBlock(runtime, element, configuration) {
                 runOnKey(evt, RET, showAnswer);
             });
 
+            // We need to register both mousedown and click event handlers because in some browsers the blur
+            // event is emitted right after mousedown, hiding our button and preventing the click event from
+            // being emitted.
+            // We still need the click handler to catch keydown events (other than RET which is handled below),
+            // since in some browser/OS combinations some other keyboard button presses (for example space bar)
+            // are also treated as clicks,
+            $element.on('mousedown click', '.go-to-beginning-button', onGoToBeginningButtonClick);
+            $element.on('keydown', '.go-to-beginning-button', function(evt) {
+                runOnKey(evt, RET, onGoToBeginningButtonClick);
```

Comment:
@haikuginger I decided to leave it as it is for now, because the surrounding code uses the same formatting. We use a mixture of different formatting styles. It would be great if we could make that more consistent at some point.